### DISCUSSION TITLE
v0 Shadcn block install

### DISCUSSION THREAD

**Discussion by janbaz497** [shadcn-ui/ui#4947] [General]
Hello,

i use v0 for my frontend and i want to install the code in my project with the block install...
Everytime i get the followed message in my terminal:

npx shadcn@latest add "https://v0.dev/chat/b/b_rQFj0C5?token=eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..OAT_VNvK7inE0zb_.LT8h992-nSXvGJsJ-wLDqSvXfJaPNpYbVXNQ6jlINaWMbbUxiYArNQ.2Zzs-efw9Vcszp0hmROVcA"
⠏ Checking registry.

Something went wrong. Please check the error below for more details.
If the problem persists, please open an issue on GitHub.

The component at https://ui.shadcn.com/r/styles/default/@/components/ui.json was not found.
It may not exist at the registry. Please make sure it is a valid component.

Iam new in programming and i dont know the problem.

I am very thankful for help

Greetings 
Jan

**Comment by abhishek207** [comment]
It is most likely happening because the dependencies version that you are using in your local codebase is different from what V0dev is using in their component codebase. So open both package.json files. One from V0dev you most likely will find in the code section, and then open your package.json in your local codebase project as well and just match the projects. Most likely it is either because of the Shadcn and UI version or the Tailwind CSS version. Just make sure you have the same versions of all the dependencies. If not, then you can either choose to upgrade or downgrade depending upon which version is more stable.
